assets/wizards/newspack/views/settings/social/meta-pixel.tsx

```diff
@@ -0,0 +1,62 @@
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+import { PAGE_NAMESPACE } from '../constants';
+import { TextControl } from '../../../../../components/src';
+import WizardsToggleHeaderCard from '../../../../wizards-toggle-header-card';
+
+const MetaPixel = () => {
+	return (
+		<WizardsToggleHeaderCard< PixelData >
+			title={ __( 'Meta Pixel', 'newspack-plugin' ) }
+			namespace={ `${ PAGE_NAMESPACE }/social/pixels/meta` }
+			description={ __(
+				'Add the Meta pixel (formerly known as Facebook pixel) to your site.',
+				'newspack-plugin'
+			) }
+			path="/newspack/v1/wizard/newspack-settings/social/meta_pixel"
+			defaultValue={ {
+				active: false,
+				pixel_id: '',
+			} }
+			fieldValidationMap={ [
+				[
+					'pixel_id',
+					{
+						callback: 'isIntegerId',
+					},
+				],
+			] }
+			renderProp={ ( { settingsUpdates, setSettingsUpdates, isFetching } ) => (
+				<TextControl
+					value={ settingsUpdates?.pixel_id ?? '' }
+					label={ __( 'Pixel ID', 'newspack-plugin' ) }
+					type="number"
```

Comment:
Can Meta pixels contain non-alphanumeric characters? The `type="number"` means that if I enter some other character it shows the `Value cannot be empty!` error:

<img width="1059" alt="Screenshot 2024-07-30 at 10 42 33 AM" src="https://github.com/user-attachments/assets/cde81e23-34fb-4a7f-8bd9-8709974a8ecf">